docs/reference/snapshot-restore/apis/create-snapshot-api.asciidoc

```diff
@@ -102,10 +102,25 @@ The cluster state includes:
 * Legacy index templates
 * Ingest pipelines
 * {ilm-init} lifecycle policies
+* Data stored in system indices, such as Watches and task records (configurable via `feature_states`)
 --
 +
 IMPORTANT: By default, the entire snapshot will fail if one or more indices included in the snapshot do not have all primary shards available. You can change this behavior by setting <<create-snapshot-api-partial,`partial`>> to `true`.
 
+[[create-snapshot-api-feature-states]]
+`feature_states`::
+(Optional, array of strings)
+A list of feature states to be included in this snapshot. A list of features
+available for inclusion in the snapshot and their descriptions be can be
+retrieved using the <<get-snapshottable-features-api,get snapshottable features API>>.
+Each feature state includes one or more system indices containing data necessary
+for the function of that feature. Providing an empty array will include no feature
```

Comment:
I have concerns regarding the consistency of what an empty array means. For indices within a CreateSnapshotRequest, it means all open indices. The behavior for features states is different and this feels wrong to me.

Reply:
That's fair and a good point - my intent with making it that way was to be able to say "I want to restore my cluster state, and no system indices". Do you have any suggestions for an alternate way of doing that?

Alternatively, is that even something we need at all? This feels like something that should be possible given you can omit features, but I don't necessarily have a use case in mind.

Reply:
A few items that I can think of would be introducing a special `none` option, completely omitting `feature_states`, or `+` `-` and `*` support.

Regarding whether we need it, I guess there could be a case where someone just wants to restore how they did previously and only include items from the cluster state.

Reply:
I agree that the `none` option is the best and also agree that it must be the only item in the array in that case.